### ISSUE TITLE
[GAME DOES NOT START]

### ISSUE THREAD

**Issue by garcianico317** [Electroblob77/Wizardry#800]
Minecraft version: 1.12.2
Forge 14.23.5.2860
Wizardry version: 4.3.6 
Environment: Singleplayer

Issue details: [the launcher can not start the game when I put this mod on my modpack, and cannot start when I use only this]

Other mods involved: [only trying to play with Electroblob's Wizardry]

Crash report 
 ---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 4/12/22 7:32 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Electroblob's Wizardry (ebwizardry)
Caused by: java.lang.NullPointerException
	at electroblob.wizardry.client.gui.handbook.GuiWizardHandbook.getHandbookResource(GuiWizardHandbook.java:466)
	at electroblob.wizardry.client.gui.handbook.GuiWizardHandbook.loadHandbookFile(GuiWizardHandbook.java:409)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:121)
	at electroblob.wizardry.client.ClientProxy.registerResourceReloadListeners(ClientProxy.java:167)
	at electroblob.wizardry.Wizardry.preInit(Wizardry.java:130)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:629)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_311, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 450135648 bytes (429 MB) / 939524096 bytes (896 MB) up to 5368709120 bytes (5120 MB)
	JVM Flags: 8 total; -Xmx5G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.5.2860 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State | ID         | Version      | Source                                      | Signature                                |
	|:----- |:---------- |:------------ |:------------------------------------------- |:---------------------------------------- |
	| LCH   | minecraft  | 1.12.2       | minecraft.jar                               | None                                     |
	| LCH   | mcp        | 9.42         | minecraft.jar                               | None                                     |
	| LCH   | FML        | 8.0.99.99    | forge-1.12.2-14.23.5.2860.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCH   | forge      | 14.23.5.2860 | forge-1.12.2-14.23.5.2860.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCE   | ebwizardry | 4.3.6        | ElectroblobsWizardry-4.3.6-MC1.12.2 (1).jar | None                                     |

	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.6.14800 Compatibility Profile Context 22.4.1 30.0.15021.1001' Renderer: 'Radeon RX 570 Series'

**Comment by WinDanesz** [comment]
What is your OS language and minecraft client language?

**Comment by garcianico317** [comment]
Hello, english both

**Comment by WinDanesz** [comment]
Probably a duplicate of #764
I'll try to replicate this issue

**Comment by judgekreep** [comment]
Hello, I used to use TLauncher and this mod worked fine.
I recently moved to use the official Minecraft launcher and started receiving the crash mentioned here.

(I just tested TLauncher just now and it works fine again - must be a launcher related issue).
Both my system language and game language are set to English.

Edit:
First, I change my Java to version 1.8.0_321 - - the crash persisted.
Next, I ran Vanilla 1.12.2, then re-installed Forge 1.12.2 - - it seemed that this fixed it for me.

I've no idea what exactly fixed it or why TLauncher worked initially and why the other Launcher didn't - perhaps I was missing a library?

1.12.2-forge-14.23.5.2860
